.github/workflows/build-and-test-snap.yml

```diff
@@ -13,8 +13,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-      - name: Checkout code
+      - name: Checkout code and download lfs dependencies
         uses: actions/checkout@v4
+        with:
+          lfs: true
+          
+      - name: Checkout LFS objects
+        run: git lfs checkout
```

Comment:
Download LFS dependencies and checkout LFS objects, as [an LFS object](https://github.com/MonicaisHer/matter-snap-testing/blob/82554e3f715693a5355f4024b2fb352e1abfdb6d/test/suites/chip-tool/chip-all-clusters-minimal-app-commit-1536ca2) has been added to the chip-tool test suite.

Reply:
These are ineffective because they are being added to the `build` job for the chip-tool-snap repo! The `build` job runs on an ephemeral instance, separate from the instance that runs the tests.

Reply:
They have been moved to the `test` job for matter-snap-testing repo now: https://github.com/MonicaisHer/matter-snap-testing/pull/2#discussion_r1447543700

Reply:
I see. I attempted to relocate these to the location where the test suite had been called, by moving them to 'matter-snap-testing' test action, right before the step ["run smoke tests"](https://github.com/MonicaisHer/matter-snap-testing/blob/IENG-881/test/action.yml#L49). However, this didn't work as it threw an error saying "[Not in a Git repository](https://github.com/canonical/chip-tool-snap/actions/runs/7476649215/job/20348440960#step:2:11)".

Do you have any ideas about this?

Reply:
Please see my answer at https://github.com/MonicaisHer/matter-snap-testing/pull/2#discussion_r1450057368